### ISSUE TITLE
LUA PHP Extension

### ISSUE THREAD

**Issue by ThomasCr** [oerdnj/deb.sury.org#2076]
Hi,

is it possible to create a DEB Package for the LUA extension?

I read, that there was previously a request for PHP 8 - but to this time there was no support.

https://github.com/oerdnj/deb.sury.org/issues/1997

But now it is possible to build it for PHP 8 - I also build it manually for PHP 8.2.

I need it for the NC plugin [files_scripts](https://github.com/Raudius/files_scripts/tree/master?tab=readme-ov-file#recommended).

I build it like that and it worked fine.

`phpize8.2 && ./configure --with-lua-version=5.3 --with-php-config=/usr/bin/php-config8.2`

Love to see the plugin in your PPA.

Thanks you in Advance

**Comment by ernolf** [comment]
> phpize8.2 && ./configure --with-lua-version=5.3 --with-php-config=/usr/bin/php-config8.2

Hi @ThomasCr:

Look [-> here <-](https://help.nextcloud.com/t/nextcloud-28-with-php8-3-on-debian-ubuntu-missing-modules-solution/174766), the php-module-builder script for Nextcloud builds the module version 2.0.7 (with lua version 5.4):

`php-module-builder 8.2 lua`

ernolf

**Comment by ThomasCr** [comment]
Hi @ernolf 

Thanks for your Information - but I already have build it - but thats not the point for this issue.... 😉

It would be perfect to add the lua extension to the PHP PPA from @oerdnj to keep it automatically updated 👍🏼.

**Comment by oerdnj** [comment]
Yeah, but this should be resolved first: https://github.com/badoo/php-lua/issues/11
